jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -1041,7 +1049,7 @@ public Field getDeclaredField(String name) throws NoSuchFieldException, Security
  */
 private native Field[] getDeclaredFieldsImpl();
 
-/*[IF Sidecar19-SE]
+/*[IF JAVA_SPEC_VERSION > 8]*/
```

Comment:
To be consistent pls use >= 11 instead of > 8. I don't see > 8 used anywhere.

Reply:
Updated.